DeveloperQuickStart.md

```diff
@@ -0,0 +1,319 @@
+If the script reports success, it indicates to the developer that any changes made have not broken any functionality across all the Ciao components.
+
+Meeting the goal originally outlined at the top of the page, build/setup/running your cluster all-in-one all transpires quickly and easily from the single script.  The time needed for ./setup.sh and ./verify.sh to build ciao from source, configure it components into a virtual cluster, then launch and teardown containers and VMs is on the order of one minute total elapsed time.
+
+## Ongoing Usage
+
+Once it's finished, the ```setup.sh``` script leaves behind a virtual cluster which can be used to perform manual tests.  These tests are performed using the [ciao-cli](https://github.com/01org/ciao/blob/master/ciao-cli/README.md) tool.  The ciao-cli tool requires that some environment variables be set up before it will work properly.  These variables contain the URLs of the various ciao services and the credentials needed to access these services.  The setup.sh script creates a shell source that contains valid values for the newly set up cluster.  To initialise these variables you just need to source that file, e.g,
+
+```
+. ~/local/demo.sh
+```
+
+To check everything is working try the following command
+
+```
+ciao-cli workload list
+```
+
+# Running ciao-webui
+
+The easiest way to develop and run the ciao-webui is inside a VM built by ciao-down.
+Not only does ciao-down download the web-ui code for you but it also downloads and
+configures all of the dependencies needed to develop the ciao-webui, such as npm.
+
+To run the webui in a ciao-down VM simply execute the following commands
+
+```
+cd ~/ciao-webui
+./deploy.sh production --config_file=$HOME/local/webui_config.json
+```
+
+And then point your host's browser at https://localhost:3000.  The
+certificate used by the web-ui inside ciao-down is self signed so you
+will need to accept the certificate in your browser before you can view
+the web-ui.
+
+Ciao-down configures an account for you to use.  The user name is 'csr'
+and the password is 'hello'.  Enter these credentials in the login screen
+to start managing your cluster.
+
+# Running the BAT tests
+
+The ciao project includes a set of acceptance tests that must pass before each release is made.  The tests perform various tasks such as listing workloads, creating and deleting instances, etc.  These tests can be run inside the machine
+
+```
+# Source the demo.sh file if you have not already done so
+. ~/local/demo.sh
+cd $GOPATH/src/github.com/01org/ciao/_release/bat
+test-cases -v ./...
+```
+
+For more information on the BAT tests please see the [README](https://github.com/01org/ciao/blob/master/_release/bat/README.md).
+
+# Cleanup / Teardown
+
+To cleanup and tear down the cluster:
+```
+cd $GOPATH/src/github.com/01org/ciao/testutil/singlevm
+#Cleanup any previous setup
+./cleanup.sh
+```
+
+# Known Issues with Bare Metal
+
+- Does not work on Fedora due to default firewall rules. 
```

Comment:
since we're noting "known issues" for specific systems we could add a note for ArchLinux:

install the package: qemu-block-rbd

othrerwise ceph storage will not work.